code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1411,9 +1411,84 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	if(user == target)
 		return FALSE
 	if(user.loc == target.loc)
+		if(user.pulling == target && user.zone_selected == BODY_ZONE_HEAD && user.grab_state >= GRAB_NECK)
```

Comment:
@Qustinnus check out this cool new intent combo

Reply:
This malarkey needs a proper framework